qrcode/android/src/main/kotlin/com/example/qrcode/QrcodePlugin.kt

```diff
@@ -54,7 +54,7 @@ class QrcodePlugin: FlutterPlugin, MethodCallHandler, ActivityAware {
   companion object {
     @JvmStatic
     fun registerWith(registrar: Registrar) {
-      registrar.platformViewRegistry().registerViewFactory("plugins/qr_capture_view", QRCaptureViewFactory(registrar, null, registrar.activity(), registrar.messenger()))
+      registrar.platformViewRegistry().registerViewFactory("plugins/qr_capture_view", QRCaptureViewFactory(registrar, null, registrar.activity()!!, registrar.messenger()))
```

Comment:
this might be looks better and safer 

          registrar.activity().let {  registrar.platformViewRegistry().registerViewFactory("plugins/qr_capture_view", QRCaptureViewFactory(registrar, null, it, registrar.messenger()))